kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/ServicesUi.ui.xml

```diff
@@ -50,6 +50,9 @@
     </ui:style>
 
     <b:Container fluid="true">
+		<b:Row>
```

Comment:
The formatting seems odd here

Reply:
I see it properly in my eclipse

Reply:
Maybe a configuration problem related to spaces and tabs?

Reply:
yes, i replaced tabs with whitespaces